### DISCUSSION TITLE
Community Wiki

### DISCUSSION THREAD

**Discussion by Root3287** [ArthurSonzogni/FTXUI#480] [General]
So far this is the best project on making a C++ TUI, but the issue is kinda the lack of information with this library. Sure there is documetnation, but certain things are lacking there. Would it be possible to open a github wiki for the community to add snippets, examples, and code to help people getting started?

**Comment by ArthurSonzogni** [comment]
Thanks!
I try to do my best with the documentation, the tutorial, the examples. I was hoping it to be good enough.

I was hoping folks to contribute directly into the documentation in the git repository, but I agree a wiki is probably more welcoming.

I just opened the wiki and allowed everyone to add pages. Let's see how it goes ;-)
Please let me know if I can do something else.

**Reply by Root3287** [reply]
Thank you!
It looks like the Wiki is not editable. I have linked a way to fix this
https://docs.github.com/en/communities/documenting-your-project-with-wikis/changing-access-permissions-for-wikis

**Reply by ArthurSonzogni** [reply]
Should be fixed now. Thanks!

**Comment by cosargozukirmizi** [comment]
I have used ftxui to build an interface for the probabilistic evolution theory solution of initial value problem of ordinary differential equations with multinomial right-hand-side functions. 
https://github.com/cosargozukirmizi/cli-prevth
